util/patch/patch_test.go

```diff
@@ -370,7 +370,7 @@ var _ = Describe("Patch Helper", func() {
 		obj := &clusterv1.Cluster{
 			ObjectMeta: metav1.ObjectMeta{
 				GenerateName: "test-",
-				Namespace:    "default",
+				Namespace:    "test-namespace",
```

Comment:
Why did you need to change the ns?

Reply:
The previous name was failing, and that was suggested by tests I remember